components/procore/actions/create-incident/create-incident.mjs

```diff
@@ -0,0 +1,103 @@
+import app from "../../procore.app.mjs";
+
+export default {
+  key: "procore-create-incident",
+  name: "Create Incident",
+  description: "Create a new incident. [See the documentation](https://developers.procore.com/reference/rest/incidents?version=latest#create-incident).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    companyId: {
+      propDefinition: [
+        app,
+        "companyId",
+      ],
+    },
+    projectId: {
+      optional: false,
+      propDefinition: [
+        app,
+        "projectId",
+        ({ companyId }) => ({
+          companyId,
+        }),
+      ],
+    },
+    title: {
+      type: "string",
+      label: "Title",
+      description: "Incident Title",
+    },
+    eventDate: {
+      type: "string",
+      label: "Event Date",
+      description: "Iso8601 datetime of Incident occurrence. If time is unknown, send in the date at `0:00` project time converted to UTC. Eg. `2025-04-01T00:00:00Z`.",
```

Comment:
```suggestion
      description: "Iso8601 datetime of Incident occurrence. If time is unknown, use the date at `0:00` project time converted to UTC. Eg. `2025-04-01T00:00:00Z`.",
```
This description is a bit unclear to me - do you think it's clear to the user given the app's context?